lib/puppet/type/wls_datasource/connectionreservetimeoutseconds.rb

```diff
@@ -0,0 +1,12 @@
+newproperty(:connectionreservetimeoutseconds) do
+  include EasyType
+
+  desc 'The number of seconds after which a call to reserve a connection from the connection pool will timeout.'
+
+  defaultto '10'
```

Comment:
@biemond 
We really *shouldn't* be providing defaults for properties like this.  It should be possible to only specify the properties we want to manage in our manifests (or when using puppet resource, eg via mcollective).

Anybody who has manually modified this datasource attribute (because the type originally didn't support it) may be surprised if puppet starts reverting their changes even though they didn't ask it to.

If a default *is* needed for creating the resource, it should be in the create.py, not in the type.